contracts/interchain-token-service/src/contract.rs

```diff
@@ -1,40 +1,32 @@
+use crate::abi::{get_message_type, MessageType as EncodedMessageType};
```

Comment:
does cargo fmt not handle this ordering automatically? Or is it an editor specific setting?

Reply:
I'll look into it